lib/ret/hub.ex

```diff
@@ -897,6 +897,57 @@ end
 
 defimpl Canada.Can, for: Ret.Account do
   alias Ret.{Hub, AppConfig}
+  alias Ret.Api.{Scopes, Credentials}
+
+  # Non-admin accounts can create their own account tokens
+  def can?(%Ret.Account{account_id: account_id}, :create_credentials, %{
+        subject_type: :account,
+        scopes: scopes,
+        account_id: account_id
+      }) do
+    # TODO: Is it correct to disallow the create_accounts scope for account type tokens?
```

Comment:
I got it into my head that only `:app` credentials would be allowed to have the `create_accounts` scope, but then I don't have a good reason for this. Maybe only admin accounts should be allowed to create `:account` credentials that have the `create_accounts` scope. Then the only important distinction between `:app` credentials and `:account` credentials is the fact that `:app` credentials are accessible / managed / shared by all admin accounts.